scrapy/extensions/feedexport.py

```diff
@@ -191,6 +191,8 @@ def __init__(self, settings):
         self.urifmt = settings['FEED_URI']
         if not self.urifmt:
```

Comment:
I would avoid repeating both `settings['FEED_URI']` and `self.urifmt` in these lines.

You could probably switch to:

```python
if not settings['FEED_URI']:
    raise NotConfigured
self.urifmt = str(settings['FEED_URI'])
```